units/host_monitoring_external_termination.go

```diff
@@ -168,3 +167,61 @@ func handleExternallyTerminatedHost(ctx context.Context, id string, env evergree
 		return false, errors.Errorf("unexpected host status '%s'", cloudStatus)
 	}
 }
+
+// handleTerminatedHostSpawnedByTask re-creates a new intent host if possible when this host.create host fails;
```

Comment:
I would put a period at the end of this sentence, or start L172 without capitalization.

Reply:
done